mitodl/micromasters
# conditionally hide course progress

#### What are the relevant tickets?

closes #3313 

#### What's this PR do?

This adds a boolean column to the `Course` table called `should_display_progress`, which governs whether course runs for that course should report progress before they have a finalized grade or not.

#### How should this be manually tested?

The column has a default value of `True`, so you should see no changes initially after running the migration. Then, if you have an in-progress course which is reporting a 'current_grade' on one of it's course runs, you should stop seeing that information in the UI if you switch `should_display_progress` to be `False`.
